core/tee/tee_svc_cryp.c

```diff
@@ -1121,17 +1259,20 @@ static TEE_Result tee_svc_cryp_check_attr(enum attr_usage usage,
 	 * that the same attribute isn't repeated.
 	 */
 	for (n = 0; n < attr_count; n++) {
-		int idx =
-		    tee_svc_cryp_obj_find_type_attr_idx(attrs[n].attributeID,
+		idx = tee_svc_cryp_obj_find_type_attr_idx(
+							attrs[n].attributeID,
 							type_props);
-		if (idx >= 0) {
-			uint32_t bit = 1 << idx;
 
-			if ((attrs_found & bit) != 0)
-				return TEE_ERROR_ITEM_NOT_FOUND;
+		/* attribute not defined in current object type */
+		if (idx < 0)
+			return TEE_ERROR_ITEM_NOT_FOUND;
```

Comment:
`continue` instead of `return` to have the same behavior than in the previous code.